src/core_codemods/fix_float_equality.py

```diff
@@ -0,0 +1,98 @@
+import libcst as cst
+
+from codemodder.codemods.libcst_transformer import (
+    LibcstResultTransformer,
+    LibcstTransformerPipeline,
+)
+from codemodder.codemods.utils_mixin import NameAndAncestorResolutionMixin
+from core_codemods.api import Metadata, Reference, ReviewGuidance
+from core_codemods.api.core_codemod import CoreCodemod
+
+
+class FixFloatEqualityTransformer(
+    LibcstResultTransformer, NameAndAncestorResolutionMixin
+):
+    change_description = "Replace `==` or `!=` with `math.isclose` callz"
```

Comment:
Sorry I missed this one on the original review.
```suggestion
    change_description = "Replace `==` or `!=` with `math.isclose`"
```